samples/system-test/topics.test.ts

```diff
@@ -100,6 +100,22 @@ describe('topics', () => {
     assert.strictEqual(receivedMessage.data.toString(), expectedMessage.data);
   });
 
+  it('should publish with flow control', async () => {
+    const [subscription] = await pubsub
+      .topic(topicNameThree)
+      .subscription(subscriptionNameOne)
+      .get({autoCreate: true});
+    const output = execSync(
+      `${commandFor('publishWithFlowControl')} ${topicNameThree}`
+    );
+    const receivedMessage = await _pullOneMessage(subscription);
+    assert.strictEqual(receivedMessage.data.toString(), 'test!');
+    assert.ok(output.indexOf('Published 1000 with flow control settings') >= 0);
+
+    // Junk any remaining published messages.
+    await subscription.seek(new Date(Date.now()));
```

Comment:
I think this is the same as `new Date()` right?

Reply:
Probably! :) I'm used to moment now...